docs/docs/configuration/server.rst

```diff
@@ -229,6 +229,86 @@ timeout
     specify a timeout in seconds for blocking operations like the
     connection attempt.
 
+Mail
+----
+
+configure the format the mail notification uses in SMTP section.
+
+.. code-block:: ini
+
+    [mail]
+    language = en
+    format = plain
+    title_admin = {title}
+    title_user = Re: New comment posted on {title}
+    template =
+
+language
+	specify the language of the mail format by its `ISO 639-1<https://en.wikipedia.org/wiki/ISO_639-1>`_ (two letter)
+	code, default to `en`. The "Anonymous" will be translated in the language you
+	set. If you use the default template, the template will also be translated into
+	the language (Note that the default templates supported to be translated using
+	the option now includes: `en`, `de`, `fr`, `ja`, `zh`, `zh-CN`, `zh-TW`. You are welcome to
```

Comment:
"Note that the default templates support the following languages:"

But please don't list the languages here, since that means we'll have to keep this list synced with the actual supported languages.